src/components/Phrase/Phrase.js

```diff
@@ -0,0 +1,36 @@
+import './Phrase.css';
+
+import React from 'react';
+
+export default function Phrase({ setPhrase, phrase, phraseArray, setPhraseArray }) {
+  const logPhrase = () => {
+    setPhraseArray((prevState) => [...prevState, phrase]);
+  };
```

Comment:
In our logPhrase function, we can call setPhrase('') which will reset the input field to be an empty string after the user clicks the 'Add' button.